### PR TITLE
Astro release notes - August 17

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Comment by chomipi88** [comment]
I think we should also update the `Log into Astro` doc to explain how Org Switching works.
Users will still be logged into their first / primary Organization. Once logged in they can use the Org Switcher to get into another Organization.

Also to note that having an Organization role in another Organization doesn't guarantee you can log into it. The other Organization needs to have a connection (eg. Basic Auth or Google) you can successfully authenticate with to log into that Organization.
If say an Astronomer employee was invited to a Customer Organization that **only** had their Okta IdP connection enabled, the Astronomer employee would not be able to authenticate to login.

We have plans to address this limitation.

File: astro/log-in-to-astro.md [hunk 1]
```diff
@@ -73,3 +73,14 @@ For Astro users, the base domain is `cloud.astronomer.io`. For Astronomer Softwa
...
     astro context switch <basedomain>
     ```
+
+## Switch Organizations
+
+You can belong to more than one Astro Organization. To switch to another Organization that you have access to:
+
+1. Log in to the Cloud UI. By default, the Cloud UI opens the first Organization that you joined.
+2. In the top navigation bar, click the name of your current Organization.
+3. Click **Change Org**.
```
Comment (chomipi88) [inline astro/log-in-to-astro.md:83]:
```suggestion
3. Click **Switch Organization**.
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,45 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))
 
+## August 18, 2022
+
+### Save on Deployment resource usage with worker queues
```
Comment (paolaperaza) [inline astro/release-notes.md:20]:
```suggestion
### Optimize your Deployment for different tasks with worker queues
```

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,47 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.3.0 ([Release notes](cli/release-notes.md))
 
+## August 18, 2022
+
+### Worker queues
+
+Worker queues are a new way to size your workers on Astro and configure your Deployment to best fit the needs of your tasks. A worker queue is a set of configurations that apply to a group of workers in your Deployment. Within a worker queue, you can configure worker type and size as well as autoscaling behavior.
+
+In the Cloud UI, you can now size workers more simply and create multiple worker queues.
```
Comment (jwitz) [inline astro/release-notes.md:24]:
```suggestion
In the Cloud UI, you can now scale workers more efficiently by creating multiple worker queues.
```

**Pull request merged**